tests/test_eks/test_eks.py

```diff
@@ -0,0 +1,770 @@
+from __future__ import unicode_literals
+
+from copy import deepcopy
+from datetime import datetime
+
+import boto3
+import mock
+import pytest
+import sure  # noqa
+
+from moto.eks.exceptions import (
+    ResourceNotFoundException,
+    ResourceInUseException,
+    InvalidRequestException,
+    InvalidParameterException,
+)
+from moto.eks.models import (
+    NODEGROUP_EXISTS_MSG,
+    CLUSTER_NOT_READY_MSG,
+    CLUSTER_NOT_FOUND_MSG,
+    CLUSTER_IN_USE_MSG,
+    LAUNCH_TEMPLATE_WITH_DISK_SIZE_MSG,
+    LAUNCH_TEMPLATE_WITH_REMOTE_ACCESS_MSG,
+    NODEGROUP_NOT_FOUND_MSG,
+    CLUSTER_EXISTS_MSG,
+)
+from moto.utilities.utils import random_string
+from test_eks_constants import (
```

Comment:
`tests/test_eks/__init__ ` should be renamed to `tests/test_eks/__init__.py` in order for pytest to play nice.
After that, the imports should be relative, with a dot preceding the files: `from .test_eks_constants import..`, `from .test_eks_utils import ..`

Reply:
ACK, thanks.